src/main/java/com/sonyericsson/rebuild/Rebuilder.java

```diff
@@ -48,15 +48,15 @@ public Rebuilder() {
 
     @Override
     public void onCompleted(Run build, TaskListener listener) {
-            for (RebuildValidator rebuildValidator : Hudson.getInstance().
-                    getExtensionList(RebuildValidator.class)) {
-                if (rebuildValidator.isApplicable(build)) {
-                    return;
-                }
+        for (RebuildValidator rebuildValidator : Jenkins.getInstance().
+                getExtensionList(RebuildValidator.class)) {
```

Comment:
no need in line break

Reply:
The original was in 2 lines but I merged it to 1